packages/oui-dual-list/README.md

```diff
@@ -0,0 +1,163 @@
+# Dual List
+
+<component-status cx-design="complete" ux="complete"></component-status>
+
+## Usage
+
+### Basic (Empty component)
+
+```html:preview
+<oui-dual-list></oui-dual-list>
+<div style="margin-bottom: 20px;"></div>
```

Comment:
Careful with this kind of inline style. It will be present in the code preview, for the copy/paste.
We have helper classes for these use cases (see `oui-doc-preview-only` and `oui-doc-preview-only-keep-children`).

But I think we could set the margin directly to `.oui-dual-list`, since it's a block component.